sagemaker-triton/fil_ensemble/2_triton_xgb_fil_ensemble.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>Confirming that fetching this image path is not supported in the SageMaker SDK, so there is no way to avoid hard-coding all these region account IDs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Confirmed.